libs/eventually-broker/src/routes/schemas.ts

```diff
@@ -1,18 +1,26 @@
 import joi from "joi";
+import { isValidCron } from 'cron-validator'
 import { Service, Subscription } from "../types";
 import * as regex from "./regex";
 
 const pgChannel = joi.string().trim().max(100).regex(regex.pg_channel);
 const outboxChannel = joi.string().trim().max(100).regex(regex.outbox_channel);
-const cronChannel = joi.string().trim().max(100).regex(regex.cron_channel);
 const voidChannel = joi.string().trim().max(100).regex(regex.void_channel);
 
+// TODO: Have a custom implementation to run macros in a more flexible way
```

Comment:
I still believe having macros would help users to easy use the cron instead of figuring out the cron expression. However using the macros alone is not sufficient. We will need to make it more flexible like how you defined the regex. We will also need to make the parser flexible for most outcomes with the macros

Reply:
I can work on this iteratively